ansible/Vagrantfile

```diff
@@ -0,0 +1,23 @@
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    config.vm.box = "bento/ubuntu-18.04"
+    config.ssh.insert_key = false
+    config.vm.network :private_network, ip: "192.168.33.33"
```

Comment:
Was the explicit IP configuration perhaps for when you were testing w/o the built-in provisioner? If so, is it still necessary to keep that here for now?